src/main/java/io/jenkins/plugin/commons/ErrorMessage.java

```diff
@@ -0,0 +1,86 @@
+package io.jenkins.plugin.commons;
```

Comment:
commons is overly broad (esp in the case of io.jenkins.plugin)  can you add a subpackage of `io.jenkins.plugin.auth.jwt`?
(ideally all classes in a plugin should have a single clear package root)